install/yaml/install.yaml

```diff
@@ -1343,7 +1343,7 @@ spec:
       serviceAccountName: agones-controller
       containers:
       - name: agones-controller
-        image: "gcr.io/agones-images/agones-controller:1.7.0"
+        image: "gcr.io/agones-images/agones-controller:1.7.0-rc"
```

Comment:
Instead of changing this to 1.8.0 after the 1.7 release, would it be better to use 1.8.0-dev (or something)? Then the RC CL would change dev -> rc and the release would drop the extension. WDYT?

Reply:
I'd be happy with that. Just need to tweak the [release checklist](https://github.com/googleforgames/agones/blob/master/docs/governance/release_process.md) to include the process.